redbot/cogs/mod/kickban.py

```diff
@@ -173,6 +173,18 @@ class KickBanMixin(MixinMeta):
             await ctx.send(_("I cannot do that due to discord hierarchy rules"))
             return
         audit_reason = get_audit_reason(author, reason)
+        if reason is None:
+            reason = "No reason was given."
+        self == self.bot.get_cog("Mod")
```

Comment:
What's this for? 👀

Reply:
So the embed isn't left with just a title if no reason was given.

Reply:
I meant the line that's right under the if statement:

Just bare ` self == self.bot.get_cog("Mod")` sitting on a line.